server/chat-commands/moderation.ts

```diff
@@ -15,10 +15,13 @@ import {Utils} from '../../lib/utils';
 const MAX_REASON_LENGTH = 300;
 const MUTE_LENGTH = 7 * 60 * 1000;
 const HOURMUTE_LENGTH = 60 * 60 * 1000;
+const IP_REGEX = /\b(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\b/;
```

Comment:
This is also used in `hosts.ts`; I wonder if the regex should maybe go in Utils or IPTools (or the `/markshared` code be moved to hosts).

Reply:
IPTools is probably a good place for now.

Reply:
Although yes, `/markshared` code should probably be moved to hosts.